### PR TITLE
[typo](doc)Update materialized-view.md

### LINKED ISSUE #18825: doris官方文档有错误
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

doris版本1.1.5

### What's Wrong?

Screenshot_20230419_181321

### What You Expected?

这里少了关键字“TABLE”，正确的语句是：CANCLE ALTER TABLE MATERIALIZED VIEW FROM table_name  

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lljqy**
文档中“取消物化视图”的语法有误，少了个“TABLE”关键字

# Proposed changes

Issue Number: close #18825 

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by luzhijing — DISMISSED** [review]
LGTM. Thanks for your Pull Request and Welcome to Apache Doris Community.

File: docs/zh-CN/docs/advanced/materialized-view.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ MySQL [test]> desc mv_test all;
 ### 取消创建物化试图
```
Comment (BiteTheDDDDt) [inline docs/zh-CN/docs/advanced/materialized-view.md:162]:
```suggestion
### 取消创建物化视图
```

**Review by luzhijing — APPROVED** [review]
LGTM

**Comment by luzhijing** [comment]
run buildall

**Pull request merged**